doc/GoogleTextToSpeechConnector.md

```diff
@@ -6,6 +6,20 @@ _ES6 import_: `import { GoogleTextToSpeechConnector } from 'reshuffle-google-con
 
 This is a [Reshuffle](https://dev.reshuffle.com) connector that provides an Interface to the Google Text to Speech  Client.
 
+#### Example
```

Comment:
This Readme uses a different template than the recent connectors.

- the Code | npm | example missing at the top
- Connector actions section

Reply:
now I get what you did after seeing your second PR.
Connector actions section and maybe a table of contents (relatively straight forward documentation so not that helpful)